pkg/operator/controller/gatewayclass/subscription.go

```diff
@@ -55,11 +55,12 @@ func desiredSubscription(name types.NamespacedName) (*operatorsv1alpha1.Subscrip
 			Name:      name.Name,
 		},
 		Spec: &operatorsv1alpha1.SubscriptionSpec{
-			Channel:                "stable",
+			Channel:                "1.1-nightly", //"stable",
 			InstallPlanApproval:    operatorsv1alpha1.ApprovalAutomatic,
-			Package:                "servicemeshoperator",
-			CatalogSource:          "redhat-operators",
+			Package:                "sailoperator",        //"servicemeshoperator3",
+			CatalogSource:          "community-operators", //"redhat-operators",
 			CatalogSourceNamespace: "openshift-marketplace",
+			StartingCSV:            "sailoperator.v1.1.0-nightly-2025-03-05",
```

Comment:
OSSM 3.0.0TP2 still has the Istio v1alpha1 CRD, so I am using a Sail Operator nightly to get the Istio v1 CRD.  I will update this subscription once OSSM 3.0 is GA.